packages/mui-joy/src/Autocomplete/Autocomplete.test.tsx

```diff
@@ -34,6 +34,58 @@ function checkHighlightIs(listbox: HTMLElement, expected: string | null) {
   }
 }
 
+type TestOnScrollToBottom = {
+  reason: 'mouse' | 'keyboard';
+  scrollToBottomCallCount: number;
+  onScrollToBottom: ReturnType<typeof spy>;
+};
+
+function testOnScrollToBottom({
+  reason,
+  scrollToBottomCallCount,
+  onScrollToBottom,
+}: TestOnScrollToBottom) {
+  const textbox = screen.getByRole('combobox');
+  const listbox = screen.getByRole('listbox');
+
+  if (reason === 'mouse') {
+    const scrollTop = listbox.scrollHeight - listbox.offsetHeight;
```

Comment:
Can't you use `listbox.scrollTop` property directly instead of calculating?

Reply:
It is returning zero when there is no scroll bar in the other test. You can keep it like this.